src/main/java/frc/robot/Constants.java

```diff
@@ -21,9 +21,12 @@ public final static class DrivetrainConstants {
 
         public static final double DRIFT_DEADBAND = 0.1;
         public static final double ROTATION_DEADBAND = 0.006;
+        public static final double BUTTON_DEADBAND = 0.2;
```

Comment:
If you want to use a deadband (which I'm not totally sure is necessary), it should probably be much smaller than 0.2. I'd suggest somewhere closer to 0.01

Reply:
If we are pressing the button it will almost certainly go above 0.2, and I am just building in a headband so that barely pressing the button doesn't trigger the lights